core/assets/game.play

```diff
@@ -1,25 +1,46 @@
 # Hamorigami Gameplay Setup
 
-# DAY 1 - nothing much is happening here
-#
-# this is a comment
-day 1
+# day 0 - story set up
+# evening only, clear skies
 
 begin cutscene
 
 wait 2 seconds
 player spawns at 200,0
 player fades in for 2 seconds
 wait 4 seconds
-player says player.dialog.01
-player starts swiping
-player moves by -200 for 3 seconds looped
-wait 2 seconds
-player emotes with smile
-player starts swiping
+player says cutscene.day0.human.01
```

Comment:
I will probably rename this to avoid confusion.

The player is in fact kodama, and I don't think kodama is saying these sentences actually, correct?

In terms of feature, I think you want to have something like "overlay text" like in the ldjam version, instead of Kodama saying that? Or how should these things be displayed exactly?